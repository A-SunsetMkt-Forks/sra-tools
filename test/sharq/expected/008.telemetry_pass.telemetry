{
    "--readTypes": "BBT",
    "args": [
        "--debug",
        "--no-timestamp",
        "--readTypes=BBT",
        "--read1PairFiles=input/003.t_R1.fastq.gz,input/003.t2_R1.fastq,input/003.t3_R1.fastq",
        "--read2PairFiles=input/003.t_R2.fastq.gz,input/003.t2_R2.fastq,input/003.t3_R2.fastq",
        "--read3PairFiles=input/003.t_I1.fastq.gz,input/003.t2_I1.fastq,input/003.t3_I1.fastq",
        "-t",
        "./actual/008.telemetry_pass/telemetry"
    ],
    "groups": [
        {
            "defline_type": [
                "illuminaNew"
            ],
            "files": [
                "input/003.t_R1.fastq",
                "input/003.t_R2.fastq",
                "input/003.t_I1.fastq"
            ],
            "has_read_names": true,
            "is_10x": true,
            "is_early_end": false,
            "is_interleaved": false,
            "max_sequence_size": 146,
            "min_sequence_size": 18,
            "number_of_reads": 9,
            "number_of_spots": 3,
            "number_of_spots_with_orphans": 0,
            "reads_per_spot": 3
        },
        {
            "defline_type": [
                "illuminaNew"
            ],
            "files": [
                "input/003.t2_R1.fastq.gz",
                "input/003.t2_R2.fastq.gz",
                "input/003.t2_I1.fastq.gz"
            ],
            "has_read_names": true,
            "is_10x": true,
            "is_early_end": false,
            "is_interleaved": false,
            "max_sequence_size": 146,
            "min_sequence_size": 18,
            "number_of_reads": 9,
            "number_of_spots": 3,
            "number_of_spots_with_orphans": 0,
            "reads_per_spot": 3
        },
        {
            "defline_type": [
                "illuminaNew"
            ],
            "files": [
                "input/003.t3_R1.fastq.bz2",
                "input/003.t3_R2.fastq.bz2",
                "input/003.t3_I1.fastq.bz2"
            ],
            "has_read_names": true,
            "is_10x": true,
            "is_early_end": false,
            "is_interleaved": false,
            "max_sequence_size": 146,
            "min_sequence_size": 18,
            "number_of_reads": 9,
            "number_of_spots": 3,
            "number_of_spots_with_orphans": 0,
            "reads_per_spot": 3
        }
    ],
    "platform_code": 2,
    "quality_code": 33
}
