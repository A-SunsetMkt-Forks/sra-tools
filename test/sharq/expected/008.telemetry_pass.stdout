Spot: NB551628:63:HTL7VAFXY:1:11101:14600:1041
reads 3:
num:1(B)
TAGAAAAAAACTGAATATGTTTTCATAAGACGACGAAAATATTTGGCTTGGCGTTCTGTTTCCCTTTGTTGAAAATCTTCACTGCAAGTAATCAATTCATTGAAGCGGCGCACGAAAAACGCGAAAGCGTTTCACGATAAATGCGA
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEAEAAEAEEEE<AA
num:2(B)
ATGAATTGATTACTTGCAGTGAAGATTTTCAACAAAGGGAAACAGAACGCCAAGCCAAATATTTTCGTCGTCTTATGAAAACATATTCAGTTTTTTTCTATGGGCCGGTGCAGTTAATGTAGGGAAAGAGTGTACTCATAAGTGTA
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE/EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEE<EEEEAEA<EEEEEAEEE<EEEEEEAEEEEEEEEEEEAE<EEEEE/
num:1(T)
GGAGCCGTGGGTGAATAG
+
AAAAAEEEEEEEEEEEA/
Spot: NB551628:63:HTL7VAFXY:1:11101:18743:1042
reads 3:
num:1(B)
ACGATATCCAAACATTCATGAAAAGCAATTTTCATATACCTGGACAGTAGAGGTCGGAAGAGATCATTATTATACTCAAATTGTAAGAGATCTCATTGATAAAGTTGGTTTAGGTTTTTATAAATGAAGCGGCGCACGAAAAACGC
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEE<EEEEEEEEEEEEEEEEEEEEAEEEEEEE66A6<<<AAAEEEEA<A
num:2(B)
TTTATAAAAACCTAAACCAACTTTATCAATGAGATCTCTTACAATTTGAGTATAATAATGATCTCTTCCGACCTCTACTGTCCAGGTATATGAAAATTGCTTTTCATGAATGTTTGGATATCGTTGGGCCGGTGCAGTTAATGTAG
+
AAAAAEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEEEEEEEEEEEEEE<EEEEEEAEEEEEEEEEEEEEEEEEEEEEEAEEEEEE/AAAEEAEEEEEEEEEE/
num:1(T)
AATGGCAGAAATAGTTGC
+
AAAAAEEEEEEEEEEEEE
Spot: NB551628:63:HTL7VAFXY:1:11101:9057:1042
reads 3:
num:1(B)
TTAGGATATCTTCATGTTTCTTTGCGAACTCAATCATTTCTGATTTCTTTTCATCATGTGCAATTAATGCTATATTCATTATAAATCACCTCATTAAACATTATATAAGAAAAAAAGCTCTAATGAAATCATTAAAGCTATAAAAT
+
AAAAAEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAAEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEAEEEEAAEEAEEEAEEEEAEAEEEEEEEEEEEEEEEEEEAAEEAAEEAEEA6A<EEEE</
num:2(B)
ATAATAATCATGATGGTATTGGTAAATTTATTGAAAAATATATTTTATAGCTTTAATGATTTCATTAGAGCTTTTTTTCTTATATAATGTTTAATGAGGTGATTTATAATGAATATAGCATTAATTGCACATGATGAAAAGAAATC
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEAEEEAEEEEEEEEEEEEEEEEEAEEE/EEEAAEE<EEEEEEEEEE<EEEEEEAEEEEEEEEAE<EE<EAEEEEEEEEEEE
num:1(T)
GGTTCCGAGTTTATTTTT
+
AAAAAEEEEEEEEEEEEE
Spot: NB551628:63:HTL7VAFXY:1:11101:19722:1042
reads 3:
num:1(B)
TTGATGTTTCTTTAAATGAAGAACACCATCTTTTAATTCAGTAACAATTCCTTCAGCATCTTCATCACTCATTGAAGCGGCGCACGAAAAACGCGAAAGCGTTTCACGATAAATGCGAAAACGGGAACGGATGGAGGTGGAGTAAG
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE<AEEEEEEAEAEEEEEE<AEEEEEEA<EEEEEEEEEEAEE<AAEE<EEEEEEEEEEEEEEEEAEEEEEEAEEAEAEAEEEEEEEEEE
num:2(B)
ATGAGTGATGAAGATGCTGAAGGAATTGTTACTGAATTAAAAGATGGTGTTCTTCATTTAAAGAAACATCAATGGGCCGGTGCAGTTAATGTAGGGAAAGAGTGTACTCATAAGTGTAGATCTCGGTGGTCGCCGTATCATTAAAA
+
AAAAAEEEEEEEEEEEAEEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEE<EAEEEEEEEEEAAEEEEAEEAE<EEEAEEAEEEEEEEEE<EEEEEEEEEEAEEEEA<AE<A<EEEEEEEEEEE/EEEEEAEE
num:1(T)
TTCATCGTATTTGTCTTA
+
AA/AAEEEEEAE/EEEAA
Spot: NB551628:63:HTL7VAFXY:1:11101:25833:1043
reads 3:
num:1(B)
CTGGTTATTGCATAAAGATTGTCCAAGTGAAAGAAAGGAGGCAAAATAATGGAAGATAAAGTTTTACTTTCTCTACAGGATTTATCTAAAAATTTCCATGTCAACGGTGGAACATTAAAAGCTGTTAATCATGTTGAAGCGGCGCA
+
AAAAAEEEEEEEEEEEAEEEAEEEEEEEEEEEEEEE/EEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEE<EEEEEEAEEAEEEE<EEEEEEEEE/EEE/E<EEAEAEEEAE6<<EEEEEE<AAEEEEE/EEAEE<AE/66</A//
num:2(B)
ACATGATTAACAGCTTTTAATGTTCCACCGTTGACATGGAAATTTTTAGATAAATCCTGTAGAGAAAGTAAAACTTTATCTTCCATTATTTTGCCTCCTTTCTTTCACTTGGACAATCTTTATGCAATAACCAGTGGGCCGGTGCA
+
AAAAAEEEEEEEE<EEEEEEEEEEAEAAAEEE/EEEEEAEEEEEEEEEAEEEEEE<<EEEEAEAEEEEEEEEEEEEEEE<AEE6EEEEEEEE/EAEA<<EEEEEEAE/AAA/EEEEEEA<AEE//AE<EE</A//</<A/<//6<<
num:1(T)
ATAGTCAAAATCATCTTG
+
AAAAAEEEEEEEEEEEEE
Spot: NB551628:63:HTL7VAFXY:1:11101:13863:1043
reads 3:
num:1(B)
TGGATATTTATTAAATACAGAGCGTTATCAAGTTTCATTGAGGTATGACAATCAAACAACTCCTGTCATTACAAGTTCAACGACCATTGAAGATAAAGAACCCTTAGGAAGTATTCATTTAGAAAAAGAAATAGAGAGTACTATTA
+
AAAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE6EEEEEEEEEEEEEEEEEEEEEEE/EEEEEEEEEEEEEEEEEEAEEEAEEEEEEEEAEEEEEEEEAEEEEEEEEEEEAEEAEEEAAEEEE<
num:2(B)
TCCAGAGACATTTGTAATCTTTTCTCGAGCATATAATCCAAATTCTACTTGAGATAAAAAAGCATCACCAAGTTGGTGATCTGTAATAGTACTCTCTATTTCTTTTTCTAAATGAATACTTCCTAAGGGTTCTTTATCTTCAATGG
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEA<EEEEEEEEEEEEEEEAEAEAEEEEEEAEEEEEEEEEEAEEEEEEEEEEEAEAEEEEEEEEEEEEA
num:1(T)
GGCAGCGTCTGCATCCCC
+
AAA6AEEEEEEEEEEEEE
Spot: NB551628:63:HTL7VAFXY:1:11101:2560:1044
reads 3:
num:1(B)
TCTACCATAATCCTTCATCTTATACTATGCAAAATACAAAAGGATTGTTCAGTATAAAAATAGTCACCTTCCATTATTTTCTTCATTGAAGCGGCGCACGAAAAACGCGAAAGCGTTTCACGATAAATGCGAAAACGGGAACGGAT
+
AAAAAEEEEEAEEEAEEEE/AEEEEAAEEEEEEEAEEEEEEAEE/EEEAEE/E/<EEEEEEAEEEEEAE<EEEE/<EEEAAE<EE/E//E/EEAEEAEEEEEEEEEEEEAEEEE/EEEEEEEE/EEEEEA/AE<E</<EAE<A6E/
num:2(B)
ATGAAGAAAATAATGGAAGGTGACTATTTATATACTGAACAATCCTTTTGTATTTTGCATAGTATAAGATGAAGGATTATGGTAGATGGGCCGGTGCAGTTAATGTAGGGAAAGAGTGTACTCATAAGTGTAGATCTCGGTGGTCG
+
A/AAAEEEEEEEE<EEEEEEE/EEEEEEA/EE/EEAEEEEE//EE<EE///E<EEEEEEEE/EE/EEEEEAEEAAEEAE6AE/EEEE/6EEE<EEEEEE<<<A</EEE<EEEEEE<E/<A<EEE6<AE/<AEE<AA<AAEEEE6A<
num:1(T)
TGTGTCGACAATGCACTT
+
AAAAAEAEEEE66EEE/E
Spot: NB551628:63:HTL7VAFXY:1:11101:19273:1044
reads 3:
num:1(B)
AGGATGCTGTAAACGAGAGTTGAGGAGGGGTTGATATGAGTATTGGAAAAAAGCTTTTATATTTAAGACAACAAAGAGGGCTTTCTCAAGAAGAGTTAGCAAGTGCTTTACACGTTTCTCGACAAACGATTTCAAAATGGGAATCA
+
AAAAAEEEEEAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEEEEEEEEEEEEEEAEE6EEEEEEEEEEEE<EEEEEEEEEEEEEAEEEEA<AEEEAAA<AEEAA
num:2(B)
ATAACATCATTTTCATATCAGGTAAACTTAAATCTGATTCCCATTTTGAAATCGTTTGTCGAGAAACGTGTAAAGCACTTGCTAACTCTTCTTGAGAAAGCCCTCTTTGTTGTCTTAAATATAAAAGCTTTTTTCCAATACTCATA
+
AAAAAEEEEEEEEEEEEEEEEAEEEEEEAEEEEEEAEEEEEEEEE<EEEEAEEAE<AAEEEEEEEEEAAEEEEEE/EEEEEEEEEEEEEEEEAEEEEE/EE6EEEAA<<EAAEEEEEEEAEE<EAAEA</EEEEE/A6/AAEAA//
num:1(T)
AATCCTGCAACTAGATGT
+
AAA/AEEEEEEEEEEEEE
Spot: NB551628:63:HTL7VAFXY:1:11101:15271:1044
reads 3:
num:1(B)
ACCTATAAAAATGGGATTTTCTATCAAATTTATAAGTAAAAAAAATGTATGAACTAATAATTAGGTTCTTTCTTACTTTAGGATGTAAATAAAAAGGGAAGAATTAATTTCATTGATAAATTCAATGATTTTAAAACTTCCTTGTT
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE/EEEEEEEEAEEEEEEEEEEEEEEEEEEEEEAEEEEAE<EEEAEA/E/<E/E/EAEEA/<EE/EEAE/E</E/A/AEEE<EE<//E<E/AA6EE
num:2(B)
ATTGGGTAAATGAAAACAAGGAAGTTTTAAAATCATTGAATTTATCAATGAAATTAATTCTTCCCTTTTTATTTACATCCTAAAGTAAGAAAGAACCTAATTATTAGTTCATACATTTTTTTTAGTTATAAATTTGATAGAAAATC
+
AAAAAEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEAEEEE66EEEE/EEE/EEEA<6EA<EEA<E<E/EE/EAAEAAEE</EEA<EEEEAE<E/E/AAAAEE/EEE/EEE<EEE///
num:1(T)
TATGTTGGAGTTAGATGA
+
AAAAAEEEEEEEEEEEEE
